src/NServiceBus.Core/Routing/MessagingBestPractices/Validations.cs

```diff
@@ -45,7 +45,7 @@ public void AssertIsValidForPubSub(Type messageType)
 
             if (!conventions.IsEventType(messageType))
             {
-                Log.Info("You are using a basic message to do pub/sub, consider implementing the more specific ICommand and IEvent interfaces to help NServiceBus to enforce messaging best practices for you.");
+                Log.Info($"You are using a basic message to do pub/sub, consider implementing the more specific ICommand and IEvent interfaces on the type '{messageType}' to help NServiceBus to enforce messaging best practices for you.");
```

Comment:
should we maybe just display `messageType.FullName` though?

Reply:
Done